lte/gateway/c/core/oai/tasks/sgw/sgw_context_manager.cpp

```diff
@@ -157,35 +157,16 @@ sgw_cm_create_bearer_context_information_in_collection(teid_t teid) {
     return NULL;
   }
 
-  OAILOG_DEBUG(
-      LOG_SPGW_APP,
-      "sgw_cm_create_bearer_context_information_in_collection " TEID_FMT "\n",
-      teid);
-
-  bstring b = bfromcstr("pgw_eps_bearer_ctxt_info_apns");
-  new_bearer_context_information->pgw_eps_bearer_context_information.apns =
-      obj_hashtable_ts_create(32, NULL, NULL,
-                              (void (*)(void**))pgw_lite_cm_free_apn, b);
-  bdestroy_wrapper(&b);
-
-  if (new_bearer_context_information->pgw_eps_bearer_context_information.apns ==
-      NULL) {
-    OAILOG_ERROR(
-        LOG_SPGW_APP,
-        "Failed to create APN collection object entry for EPS bearer S11 "
-        "teid " TEID_FMT "\n",
-        teid);
-    spgw_free_s11_bearer_context_information(&new_bearer_context_information);
-    return NULL;
-  }
-
   /*
    * Trying to insert the new tunnel into the tree.
```

Comment:
Please update the comment

```suggestion
   * Trying to insert the new tunnel into the map.
```